tests/python/accuracy/public_scope.json

```diff
@@ -395,5 +395,16 @@
                 ]
             }
         ]
+
+    },
+        {
+        "name": "otx_models/action_cls_xd3_kinetic.xml",
```

Comment:
Please modify tests/python/accuracy/prepare_data.py accordingly to get this file downloaded